cmd/exporter/exporter.go

```diff
@@ -1,15 +1,15 @@
 package exporter
 
 import (
+	"fmt"
+	"log/slog"
 	"net/http"
 	"os"
 	"os/signal"
 	"syscall"
 
 	"github.com/app-sre/acs-cve-prom-exporter/internal/acs"
 	"github.com/app-sre/acs-cve-prom-exporter/internal/collector"
-	"github.com/go-kit/log"
-	"github.com/go-kit/log/level"
```

Comment:
should we get rid of these 2 in all our repos?

Reply:
I'd only remove them in those that it is used alongside `promlog`. Since that one has been removed, it affects our ability to maintain exporters since we have to remain in `prometheus/common` <= 0.61.0.

I see that go-kit/log is still maintained, so it's a matter of choice for other projects.

I've only found that we use promlog in aws-resource-exporter. We'll take care of that in functional team 2.